tags.json

```
@@ -2286,6 +2286,10 @@
     "weather",
     "light"
   ],
+  "sun-dim": [
+    "dim sun",
```

```suggestion
    "brightness",
```

(remove "dim sun" tag as it's not really useful, and add "brightness")

---

Thanks 🙂